### PR TITLE
Fix the Web platform team's codeowners link

### PR DESCRIPTION
**Pull request by YuriSizov**
Noticed GitHub complains about it.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -140,7 +140,7 @@ doc_classes/*                       @godotengine/documentation
 
 /platform/android/                  @godotengine/android
 /platform/ios/                      @godotengine/ios
-/platform/javascript/               @godotengine/html5
+/platform/javascript/               @godotengine/web
 /platform/linuxbsd/                 @godotengine/linux-bsd
 /platform/macos/                    @godotengine/macos
 /platform/uwp/                      @godotengine/uwp
```
Comment (akien-mga) [inline .github/CODEOWNERS:146]: Can be removed.

**Review by fire — APPROVED** [review]
Seems to fix web and removed uwp. Looks good to me.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.2.1.

**Comment by YuriSizov** [comment]
Cherry-picked for 4.1.4.

**Pull request merged**